keps/sig-network/2433-topology-aware-hints/README.md

```diff
@@ -304,17 +289,56 @@ and disabled states. Without this fallback, endpoints could easily get
 overloaded as hints were being added or removed from some EndpointSlices but
 had not yet propagated to all of them.
 
+Note: Some future heuristics may not rely on hints and could instead be
+implemented directly by kube-proxy.
+
 ### EndpointSlice Controller
 
 When the `TopologyAwareHints` feature gate is enabled and the annotation is set
-to `Auto` for a Service, the EndpointSlice controller will add hints to
-EndpointSlices. These hints will indicate where an endpoint should be consumed
-by proxy implementations to enable topology aware routing.
+to `Auto` or `ProportionalCPU` for a Service, the EndpointSlice controller will add
+hints to EndpointSlices. These hints will indicate where an endpoint should be
+consumed by proxy implementations to enable topology aware routing.
+
+## Heuristics
+
+This KEP starts with the following heuristics:
+
+* Auto: EndpointSlice controller and/or underlying dataplane can choose the
+        heuristic used.
+* ProportionalCPU: Endpoints will be allocated to each zone proportionally,
+                   based on the allocatable Node CPU in each zone.
 
-The EndpointSlice controller will determine how many endpoints should be
-available for each zone based on the proportion of CPU cores in each zone. If
-it is not possible to determine the number CPU cores, 1 core per node will be
-assumed for calculations.
+In the future, additional heuristics may be added. Until that point, "Auto" will
```

Comment:
User should be able to say Auto or whatever-we-name-the-current thing, right?

Reply:
I was thinking it would be safer to leave this unchanged until we have more than one heuristic available. Adding a second heuristic may make naming decisions a bit clearer.